src/SourceBuild/content/Directory.Build.props

```diff
@@ -49,6 +63,7 @@
     <PortableRid Condition="'$(PortableRid)' == '' and '$(TargetOS)' == 'windows'">win-$(TargetArchitecture)</PortableRid>
 
     <TargetRid Condition="'$(PortableBuild)' == 'true' and '$(PortableRid)' != ''">$(PortableRid)</TargetRid>
+    <TargetRid Condition="'$(ShortStack)' == 'true'">$(TargetOS)-$(TargetArchitecture)</TargetRid>
```

Comment:
Isn't this wrong on Windows? i.e. `win-x64` would be `windows-x64`.

Reply:
Why can't you use `PortableRid` here instead?

Reply:
The win/windows/Windows_NT split is exhausting at times.